src/cmd/report.go

```diff
@@ -111,11 +112,11 @@ type githubRelease struct {
 	Assets []githubReleaseAsset
 }
 
-func fetchGitHubData(repo string) {
+func reportGitHubData(repo string) {
```

Comment:
These functions should still be fetch as they is what they are doing. The command is reporting, not these functions.